### ISSUE TITLE
Project Title was included on User Groups

### ISSUE THREAD

**Issue by valcaro87** [projectknife/Migrator#8]
hello,

im not sure why project title were included on the "viewing access" levels..is this a bug?
![user viewing access levels](https://f.cloud.github.com/assets/5020577/846798/6ca74a80-f425-11e2-8e5c-ea61e0c152b3.jpg)

Please see attached. thanks.

**Comment by simjuande** [comment]
![acl](https://f.cloud.github.com/assets/4609552/846995/3c3338ce-f42e-11e2-9b23-2953146b01b8.jpg)

I have the same issue when I check now my ACL.

**Comment by simjuande** [comment]
@valcaro87 you already fix your problem?

**Comment by valcaro87** [comment]
@simjuande , nope. i haven't touch anything. I'm just waiting for @eaxs for his feedback about this. Thanks.

**Comment by valcaro87** [comment]
hello? any updates on this? thanks. ;)

**Comment by kyleledbetter** [comment]
Hi, have you (first backed up) and tried upgrading to either 4.2 Beta 1 http://projectfork.net/blog/13-releases/71-projectfork-4-2-beta-1-released or the latest dev build https://github.com/projectfork/Projectfork/archive/dev.zip ?

I don't think we've seen this bug anywhere else
